README.md

```diff
@@ -695,28 +277,6 @@ _31 March 2017_
 
 - 🌟 First public release!
 
-Contributing
-============
```

Comment:
Maybe a link to the docs, here, to invite people to contribute?

Reply:
yep, i forgot that!